realm-jni/src/io_realm_internal_UncheckedRow.cpp

```diff
@@ -288,3 +288,10 @@ JNIEXPORT jboolean JNICALL Java_io_realm_internal_UncheckedRow_nativeIsAttached
     TR_ENTER_PTR(nativeRowPtr)
     return ROW(nativeRowPtr)->is_attached();
 }
+
+JNIEXPORT jboolean JNICALL Java_io_realm_internal_UncheckedRow_nativeHasField
```

Comment:
Actually it does. In order to consolidate error messages you can no longer use getColumnIndex(fieldName) with illegal field names, it will throw an exception instead of returning -1. So the implicit behaviour of getColumnIndex to check for existence now require an explicit function.